packages/riipen-ui/src/components/TableColumn.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+class TableColumn extends React.Component {
+  static propTypes = {
+    /**
+     * A callback function that returns
+     * the header cell to render for the column
+     * when called with the entity
+     */
+    cell: PropTypes.func,
+
+    /**
+     * A callback function that returns
+     * the header cell to render for the column
+     */
+    header: PropTypes.func,
+
+    /**
+     * The label to display when not rendering the header
+     */
+    label: PropTypes.string
+  };
+
+  static defaultProps = {
+    cell: () => {},
+    header: () => {}
+  };
+}
+
+export default TableColumn;
```

Comment:
okok back to my comment about how we pass in column configuration:

I think this is a pink flag, a react component that doesn't do anything. We are declaring an interface: "hey this is how we will pass in table column info".  But then that component is torn apart and never rendered. I'll think on this one overnight because I do like the way passing in JSX columns looks, but it seems a bit.. not quite right.

Reply:
I am not opposed to the idea of using a node to store information used by other components to render something as that is essentially what a HOC is. But that is my opinion on this, Id love to get other people feedback. 
This allows us to pass the rendering data in the children of the Table component rather than as an additional prop. Both are valid but I prefer this way.